tensorflow/python/training/saver_test.py

```diff
@@ -696,7 +684,10 @@ def _testPartitionedVariables(self, use_resource):
     var_full_shape = [10, 3]
     # Allows save/restore mechanism to work w/ different slicings.
     var_name = "my_var"
-    saved_path = os.path.join(_TestDir("partitioned_variables"), "ckpt")
+    saved_dir = os.path.join(self.get_temp_dir(), "partitioned_variables")
```

Comment:
I think it's still worthwhile to reduce the number of repeated lines by 50%. It's going to make it easier to maintain and less error prone.